packages/odyssey-eslint/package.json

```diff
@@ -9,6 +9,7 @@
     "@typescript-eslint/eslint-plugin": "^4.31.1",
     "@typescript-eslint/parser": "^4.31.1",
     "eslint": "^7.27.0",
+    "eslint-config-prettier": "^8.3.0",
```

Comment:
to be clear, these config additions are strictly to prevent our existing linting configuration (and their associated style only rules) from conflicting with the automatically generated prettier changes